docs/source/guides/crosshairmodding.rst

```diff
@@ -0,0 +1,161 @@
+Crosshair Modding
+=====================
+
+
+This guide shows how the following example mod was made
+
+`Custom.Crosshairs <https://github.com/MysteriousRSA/Custom.Crosshairs>`__
+
+
+INSTRUCTIONS
+============
```

Comment:
These `========` lines are reserved for unique entries in the index. Please use `-`'s for subheaders and `^`'s for subsubheaders. Otherwise it shows up wrong as seen [here](https://r2northstar--4.org.readthedocs.build/en/4/guides/crosshairmodding.html)

Reply:
I made some changes, removed = headings and got rid of links for images that weren't used,

Reply:
edit, I'm an idiot that doesn't understand github, should be good now

Reply:
you removed every single `====` heading - you want to keep one for the page title.

Also, the section you added is too readme-like for me again

Reply:
really, im still not sure about this entire PRs content being about your mod instead of the concept of how a mod like this works.

Reply:
ok, I edited the heading

what do you think would work better? I'm not too sure how to change it